### DISCUSSION TITLE
What is the "pidinet" preprocessor? What model to use with it?

### DISCUSSION THREAD

**Discussion by Drakmour** [Mikubill/sd-webui-controlnet#170] [Q&A] [answered]
What is the "pidinet" preprocessor? What model to use with it?

**Comment by zhmou** [comment, marked as answer]
Searched google and found this repo:
https://github.com/zhuoinoulu/pidinet
The preprocessor is similar to the HED, according to the output image given in the README.
So I guess using HED model performs well.

**Reply by Drakmour** [reply]
> Searched google and found this repo: https://github.com/zhuoinoulu/pidinet The preprocessor is similar to the HED, according to the output image given in the README. So I guess using HED model performs well.

Yeah, I saw it too. But I thought that each preprocessor must have it's own model.
